lib/dom-parser.js

```diff
@@ -1,3 +1,8 @@
+var htmlEntity = require('./entities');
+var XMLReader = require('./sax').XMLReader;
+var DOM = require('./dom');
+var DOMImplementation = DOM.DOMImplementation;
```

Comment:
I would *personally* rather see newly added code use `const` declarations whenever possible, which should be no problem on the now minimum Node.js version of 10.

Reply:
I agree 100%, but I noticed the entire rest of the codebase was using ES5. That's why I kept it consistent.

Reply:
@esbullington I think you are right to keep it consistent at this point.